### DISCUSSION TITLE
Gist 관련하여 문의드립니다.

### DISCUSSION THREAD

**Discussion by baramoss** [pronist/hELLO#185] [Q&A] [answered]
안녕하십니까, 스킨 사용 허용 관련해서 항상 감사드립니다.

v4.1로 변경 후 Gist 붙여넣을 시 표시가 되지 않는 이슈가 있는데, 혹시 사용하려면 html 모드에서 추가해야하는 것이 있을까요?
지난해 답변해 주신 것 참조하였습니다. 현재는 가능한 방법이 있을지 문의드립니다. 감사합니다.

![image](https://user-images.githubusercontent.com/59471361/216476728-032740f7-7424-4b49-9368-1dbac2caf228.png)

![image](https://user-images.githubusercontent.com/59471361/216476741-015ad38a-1b11-41bd-bbd8-36c9cbdc4bfa.png)

**Comment by pronist** [comment, marked as answer]
@baramoss 안녕하세요.

이미지 지연 로딩 구현으로 본문의 렌더링 방식을 변경하였는데, 그것으로 인한 부작용으로 파악됩니다. 지금 단계에서 이것을 해결하기 위해서는 이미지 지연 로딩을 포기하는 방식으로 가야지 될 듯 싶네요. 다만 이러한 부작용 해결을 위해 아직까지는 현재 구현된 이미지 지연 로딩을 롤백 할 예정은 아직 없습니다. 이는 블로그의 성능과 직결되는 요소이기 때문입니다.

**Comment by baramoss** [comment]
답변 감사드립니다!
